docs/source/modelon.impact.client.experiment_definition.rst

```diff
@@ -52,6 +52,14 @@ modelon.impact.client.experiment\_definition.model\_based module
    :undoc-members:
    :show-inheritance:
 
+modelon.impact.client.experiment\_definition.modifiers module
+-------------------------------------------------------------
+
+.. automodule:: modelon.impact.client.experiment_definition.modifiers
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
```

Comment:
I don't think we should expose this module in public docs yet. When we do we should document the classes/methods.

Reply:
Good catch! :100:

Reply:
Thanks for the review @iakovn ! I removed the docs addition. Could you please take another look?